### ISSUE TITLE
Build and runtime errors in deployed Netlify site

### ISSUE THREAD

**Issue by galvarez421** [ekoeryanto/netlify-cms-widgets#48]
I am not able to get the widget running on a deployed Netlify site.

The first issue I had was that the example npm installation code (`npm install netlify-cms-widget-fontawesome react-virtualized-select @fortawesome/fontawesome @fortawesome/react-fontawesome @fortawesome/fontawesome-free-solid`) seems to omit some packages that are necessary for a Netlify deploy to work. I also had to install @fortawesome/fontawesome-free-brands, @fortawesome/fontawesome-free-regular, and @fortawesome/fontawesome-svg-core otherwise the Netlify deploy would fail with errors like this:

```
Error: ./~/@fortawesome/react-fontawesome/index.js
10:59:42 PM:   Module not found: Error: Cannot resolve module '@fortawesome/fontawesome-svg-c  ore' in /opt/build/repo/node_modules/@fortawesome/react-fontawesome
```

```
Error: ./~/netlify-cms-widget-fontawesome/dist/umd/fontawesome.min.js
10:40:03 PM:   Module not found: Error: Cannot resolve module '@fortawesome/fontawesome-free-  brands' in /opt/build/repo/node_modules/netlify-cms-widget-fontawesome/dist/um  d
```

After I installed the additional packages, the deploy would work and the widget would appear in the CMS, but when I clicked the widget to open it I would get a different error: the CMS would crash and I'd be directed to this error: https://reactjs.org/docs/error-decoder.html?invariant=130&args[]=object&args[]=

My CMS customization script looked like this:

```
import React from "react";
import CMS from "netlify-cms";
import "netlify-cms/dist/cms.css";
import * as FontawesomeWidget from "netlify-cms-widget-fontawesome";
import "react-select/dist/react-select.css";
import "react-virtualized/styles.css";
import "react-virtualized-select/styles.css";

CMS.registerWidget(
  "fontawesome-solid",
  FontawesomeWidget.Solid,
  FontawesomeWidget.Preview
);
```

**Comment by ekoeryanto** [comment]
Are you using netlify-cms 2.0?
I am not testing it with ^2.0 yet

**Comment by galvarez421** [comment]
No, I am using netlify-cms version 1.9.3 and gatsby-plugin-netlify-cms version 2.0.1.

I tried using just react-select and encountered a similar issue: the widget would appear in the CMS but when I try to open it the CMS crashes, but this time the error was https://reactjs.org/docs/error-decoder.html?invariant=44. I'm not sure if the errors related, but the behavior is definitely similar. It could be something I'm missing on my end, but I'm not sure...

**Comment by ekoeryanto** [comment]
any repo to reproduce?

**Comment by ekoeryanto** [comment]
can you try with newly released package?

**Comment by galvarez421** [comment]
Yes, I will try again this week with the new package to see if it works. If it doesn't, I'll set up a repo to reproduce.
